include/aikido/planner/ConfigurationToConfigurations.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef AIKIDO_PLANNER_CONFIGURATIONTOCONFIGURATIONS_HPP_
+#define AIKIDO_PLANNER_CONFIGURATIONTOCONFIGURATIONS_HPP_
+
+#include <unordered_set>
+#include "aikido/constraint/Testable.hpp"
```

Comment:
Not: There needs to be an empty line between aikido header and standard headers.

Reply:
Why? `clang-format` won't order them correctly if we add empty lines.

Reply:
From a recent comment by @psigen , AIKIDO follows this convention:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes